recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -1,3 +1,4 @@
+@echo on
```

Comment:
```suggestion
```

Reply:
What's the issue with having echos in the script? The output of that batfile (in a separate step on azure, no less) is currently just the fact that it was called at all - I cannot see what would be bothersome about it, and it would help debugging (I hope) like there are currently on the scipy feedstock...

Reply:
It adds to noise in the log. The way to debug issues like these is to have an internal copy like @beckermr said and debug there.

Reply:
There'd be no noise on azure (because it's a separate stage), only in the raw log...

I consider myself relatively well-versed in the cf-infra, but I'd have no idea how to configure smithy to use a local copy of the ci-script (much less that this is the recommended debugging path), so I'm pretty sure this will bite other maintainers as well.

Considering how precious cf/core-resources are (I've been trying to solve this on the scipy-feedstock for almost 2 weeks with no response), I think accepting a little noise - only affecting the raw logs - to enable non-core maintainers to debug things themselves is a much better trade-off than the status quo.

Reply:
The only output currently is `The system cannot find the file specified.` so any noise would be welcome to help debug the problem.